ios/FluentUI/Popup Menu/PopupMenuController.swift

```diff
@@ -31,6 +31,18 @@ open class PopupMenuController: DrawerController {
         static let descriptionVerticalMargin: CGFloat = 12
     }
```

Comment:
There were some talks about moving this sort of info out of token sets, so I just didn't touch these for now. I can update if needed, though.

Reply:
what do you mean? we can't use the spacing token?

Reply:
I think @mischreiber was saying we don't want to expose them anymore? I can probably update the code to just reference the values, and not add them to the token set.

Reply:
This is good for now, but funnily enough I'll probably move these values back here once I merge https://github.com/microsoft/fluentui-apple/pull/1152 and global tokens become immutable static values again.